tests/test_core.py

```diff
@@ -507,12 +508,30 @@ def test_sum(self):
             gpd_sum, dd_sum.drop(columns=["name", "iso_a3"]), check_like=True
         )
 
+    @pytest.mark.skipif(
+        Version(dask.__version__) == Version("2022.01.1"),
+        reason="Regression in dask 2022.01.1 https://github.com/dask/dask/issues/8611",
+    )
     def test_split_out(self):
         gpd_default = self.world.dissolve("continent")
         dd_split = self.ddf.dissolve("continent", split_out=4)
         assert dd_split.npartitions == 4
         assert_geodataframe_equal(gpd_default, dd_split.compute(), check_like=True)
 
+    @pytest.mark.skipif(
+        Version(dask.__version__) == Version("2022.01.1"),
+        reason="Regression in dask 2022.01.1 https://github.com/dask/dask/issues/8611",
+    )
+    @pytest.mark.xfail
+    def test_split_out_name(self):
```

Comment:
I added an equivalent test with a different geometry name (since this has come up in the discussion as a case that could fail with certain solutions). Adding this custom one for now, but we might want to see if we can improve the workflow here, eg with some parametrization on the dataset, so this can more easily be done on multiple tests.

Reply:
That is a good point. We should test renamed geometries extensively as it will likely be a pain point, given how dask treats dataframes internally.